packages/modules/workflow-engine-redis/src/utils/workflow-orchestrator-storage.ts

```diff
@@ -108,6 +119,17 @@ export class RedisDistributedTransactionStorage extends DistributedTransactionSt
     })
   }
 
+  private async executeScheduledJob(
+    jobId: string,
+    schedulerOptions: SchedulerOptions
+  ) {
+    // TODO: In the case of concurrency being forbidden, we want to generate a predictable transaction ID and rely on the idempotency
```

Comment:
I have plans to support concurrency control in the workflow, so I believe we will be able to reuse it here.
the idea is to support max concurrent executions by workflow id

Reply:
Yeah I am not planning on adding anything yet, just added it here as something we need to consider (as oftentimes, you don't want more than 1 instance of a cronjob running at the same time)